pages/common/shift.md

```diff
@@ -0,0 +1,22 @@
+# shift
+
+> A bash built-in command that shifts the (numbered) arguments of a function or script by a specified number of places.
+> See <https://www.gnu.org/software/bash/manual/bash.html> for more information.
+
+- Shift arguments by one place; argument `$1` is dropped, argument `$2` is now `$1`:
+
+```bash
+# prints two times the same output
+echo $2
+shift
+echo $1 
+```
```

Comment:
@stku1985 I'd do something like this:
```suggestion
- Move all arguments back by one (dropping the first argument):

`shift`
```

Reply:
Ok I will do so. I think this reduces the intelligibility of the page, but I also understand that you want to keep consistency between pages.

Reply:
Why does it decrease the intelligibility @stku1985?

Reply:
Well I think scripting commands (such as shift, return, continue, for, ...) need some context. For experienced bash users this short example is fine; for people new to bash scripting my original version or even a whole function example would be easier to grasp. 

But don't worry; either way is fine from my point of view.

Reply:
I agree so much with that—but that's not what tldr-pages was originally invented for.  It's really for documented syntax of a command, not really how to use it in conjunction with other commands.